v2/jwt/jwt.go

```diff
@@ -38,80 +45,54 @@ type Claims struct {
 	EnlightClaims
 }
 
-const (
-	TokenUseAccess = "access"
-	TokenUseID     = "id"
-)
-
-func (c Claims) Valid() (err error) {
-	if err = c.RegisteredClaims.Valid(); err != nil {
-		return
-	}
```

Comment:
Why removed?

Reply:
This function is removed in `v5`, and this parsing is instead done in `jwt.Parse`. Custom validation of claims is done if they implement the interface `Validate() error`.

Reply:
Cool, thanks!